src/index.ts

```diff
@@ -9,42 +9,32 @@ const run = async (): Promise<void> => {
 
   console.log('->> Locating images…')
 
-  const results: ProcessedImagesResult = await processImages()
-
-  const optimisedImages = results.images.filter(
-    img => img.compressionWasSignificant
-  )
+  const processingResults: ProcessedImagesResult = await processImages()
 
   // If nothing was optimised, bail out.
-  if (!optimisedImages.length) {
+  if (!processingResults.optimisedImages.length) {
     console.log('::warning:: Nothing left to optimise. Stopping…')
```

Comment:
Should we be using `console.warn` and `console.debug` throughout?

Reply:
It could be, though GitHub actions has it's own `::warning::` and `::debug::` log stanza so I hadn't bothered to set the levels accordingly.